app/views/mobile_dashboard/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<div ng-controller="apiCtrl" ng-init="setMobile(true)">
+  <div class="country-header">
+    <div class="country-items">
+      <%= render(:partial => "header", :controller => "dahsboard") %>
+    </div>
+    <div class="country-messages">
+      <div ng-if="controlMessage">Important Messages: {{controlMessage}}</div>
+      <%= link_to 'Logout', destroy_user_session_path, :method => :delete, :class => 'button logoff' %>
+    </div>
+  </div>
+
+  <div class="tabs">
+    <%= render(:partial => "navigation", :controller => "dahsboard") %>
+
+    <div id="tab-news" class="tab news" ng-show="focusedTab === 'news'" ng-controller="newsCtrl">
+      <%= render(:partial => "news", :controller => "dahsboard") %>
+    </div>
+
+    <div id="tab-messages" class="tab messages" ng-if="focusedTab === 'messages'" ng-controller="messagesCtrl">
+      <%= render(:partial => "messages", :controller => "dahsboard") %>
+    </div>
+
+
+    <!-- Operatives -->
+
+    <!-- Spying -->
+
+    <div id="tab-income" class="tab income" ng-if="checkPermissions('income') && focusedTab === 'income'" ng-controller="incomeCtrl">
```

Comment:
Out of curiosity, why not just use ng-show/ ng-hide and then just disable the background queries themselves check for focus?  We would save time on re-rendering the tab every time you select it.